out/main.js

```diff
@@ -0,0 +1,9 @@
use strict";var w;function o(){return w.apply(null,arguments)}function T(e){w=e}function M(e){return e instanceof Array||Object.prototype.toString.call(e)==="[object Array]"}function x(e){return e!=null&&Object.prototype.toString.call(e)==="[object Object]"}function _(e,t){return Object.prototype.hasOwnProperty.call(e,t)}function st(e){if(Object.getOwnPropertyNames)return Object.getOwnPropertyNames(e).length===0;var t;for(t in e)if(_(e,t))return!1;return!0}function R(e){return e===void 0}function $(e){return typeof e=="number"||Object.prototype.toString.call(e)==="[object Number]"}function ge(e){return e instanceof Date||Object.prototype.toString.call(e)==="[object Date]"}function Ut(e,t){var s=[],r,n=e.length;for(r=0;r<n;++r)s.push(t(e[r],r));return s}function te(e,t){for(var s in t)_(t,s)&&(e[s]=t[s]);return _(t,"toString")&&(e.toString=t.toString),_(t,"valueOf")&&(e.valueOf=t.valueOf),e}function V(e,t,s,r){return hs(e,t,s,r,!0).utc()}function Zs(){return{empty:!1,unusedTokens:[],unusedInput:[],overflow:-2,charsLeftOver:0,nullInput:!1,invalidEra:null,invalidMonth:null,invalidFormat:!1,userInvalidated:!1,iso:!1,parsedDateParts:[],era:null,meridiem:null,rfc2822:!1,weekdayMismatch:!1}}function f(e){return e._pf==null&&(e._pf=Zs()),e._pf}var rt;Array.prototype.some?rt=Array.prototype.some:rt=function(e){var t=Object(this),s=t.length>>>0,r;for(r=0;r<s;r++)if(r in t&&e.call(this,t[r],r,t))return!0;return!1};function nt(e){if(e._isValid==null){var t=f(e),s=rt.call(t.parsedDateParts,function(n){return n!=null}),r=!isNaN(e._d.getTime())&&t.overflow<0&&!t.empty&&!t.invalidEra&&!t.invalidMonth&&!t.invalidWeekday&&!t.weekdayMismatch&&!t.nullInput&&!t.invalidFormat&&!t.userInvalidated&&(!t.meridiem||t.meridiem&&s);if(e._strict&&(r=r&&t.charsLeftOver===0&&t.unusedTokens.length===0&&t.bigHour===void 0),Object.isFrozen==null||!Object.isFrozen(e))e._isValid=r;else return r}return e._isValid}function be(e){var t=V(NaN);return e!=null?te(f(t),e):f(t).userInvalidated=!0,t}var Et=o.momentProperties=[],at=!1;function it(e,t){var s,r,n,a=Et.length;if(R(t._isAMomentObject)||(e._isAMomentObject=t._isAMomentObject),R(t._i)||(e._i=t._i),R(t._f)||(e._f=t._f),R(t._l)||(e._l=t._l),R(t._strict)||(e._strict=t._strict),R(t._tzm)||(e._tzm=t._tzm),R(t._isUTC)||(e._isUTC=t._isUTC),R(t._offset)||(e._offset=t._offset),R(t._pf)||(e._pf=f(t)),R(t._locale)||(e._locale=t._locale),a>0)for(s=0;s<a;s++)r=Et[s],n=t[r],R(n)||(e[r]=n);return e}function pe(e){it(this,e),this._d=new Date(e._d!=null?e._d.getTime():NaN),this.isValid()||(this._d=new Date(NaN)),at===!1&&(at=!0,o.updateOffset(this),at=!1)}function E(e){return e instanceof pe||e!=null&&e._isAMomentObject!=null}function Ht(e){o.suppressDeprecationWarnings===!1&&typeof console<"u"&&console.warn&&console.warn("Deprecation warning: "+e)}function C(e,t){var s=!0;return te(function(){if(o.deprecationHandler!=null&&o.deprecationHandler(null,e),s){var r=[],n,a,i,h=arguments.length;for(a=0;a<h;a++){if(n="",typeof arguments[a]=="object"){n+=`
```

Comment:
I think you'd better remove this file because it's unnecessary

Reply:
As you wish.